designsafe/apps/data/templates/data/data_depot.html

```diff
@@ -12,8 +15,8 @@
     <meta name="citation_keywords" content="{{keyword}}">
     {% endfor %}
     {% for author in authors %}
-        <meta name="citation_author" content="{{author.full_name}}">
-        <meta name="citation_author_institution" content="{{author.institution}}">
+        {% if author.full_name %}<meta name="citation_author" content="{{author.full_name}}">{% endif %}
```

Comment:
For consistency's sake, can you remove these conditionals?  I'd rather know if these tags are empty.